### ISSUE TITLE
Flexible API for swipe action

### ISSUE THREAD

**Issue by retyui** [mobile-dev-inc/maestro#574]
Thanks for your wonderful effort! 
We are actively looking into using it as our primary e2e framework.

The [`swipe` action](https://maestro.mobile.dev/reference/scroll-and-swipe)  on elements is currently unstable because different devices have different screen sizes. 

we use hard-coded  cords:

```yaml
- swipe:
    start: 90,1733
    end: 800,1733
```

Just wondering how to increase stability.

**Comment by amanjeetsingh150** [comment]
Hey, @retyui we have an upcoming change that will help you to directionally swipe on the elements. For example, you can do 

```
- swipe:
    direction: LEFT
    element: 
       text: "element-text-here"
```

You can see the example here: https://github.com/mobile-dev-inc/maestro/pull/585

Will this address your use case? Or is there anything else you are looking for?

**Comment by retyui** [comment]
@amanjeetsingh150 Yes, it is what I was searching for. When will it be merged?

**Comment by amanjeetsingh150** [comment]
Hey @retyui we have merged this and it's available in latest version of cli 1.19.0. Let me know if this works.

**Comment by amanjeetsingh150** [comment]
Feel free to reopen if this doesn't work.
